app/controllers/concerns/blacklight/search_fields.rb

```diff
@@ -32,27 +32,9 @@ def search_field_list
     blacklight_config.search_fields.values
   end
 
-  # Looks up a search field blacklight_config hash from search_field_list having
```

Comment:
I think we need to preserve these methods for backwards compatibility, update them to emit a deprecation warning, and then call the new location for this behavior (if applicable).